vizro-ai/changelog.d/20240207_133932_anna_xiong_add_azure_openai.md

```diff
@@ -0,0 +1,47 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+<!--
+### Removed
+
+- A bullet item for the Removed category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Added
+
+- Add Azure OpenAI LLM model integration. ([#301](https://github.com/mckinsey/vizro/pull/301))
```

Comment:
Is this specifically about Azure or can we say something more general, llke "Add support for ABC type models that follow XYZ protocol, e.g. Azure OpenAI"?

Reply:
same as below